services/deepseek.yml

```diff
@@ -0,0 +1,5 @@
+id: deepseek
+name: DeepSeek
+rules:
+  - '||deepseek.com^'
+icon_svg: <svg xmlns="http://www.w3.org/2000/svg" fill="currentColor" viewBox="0 0 30 30"><path d="M29.683 5.816c-.318-.156-.455.141-.64.292-.064.049-.117.112-.171.17-.464.496-1.007.822-1.716.783-1.036-.058-1.921.267-2.703 1.06-.166-.977-.718-1.561-1.559-1.935-.44-.195-.885-.389-1.193-.812-.215-.301-.274-.637-.381-.968-.068-.199-.137-.404-.367-.438-.249-.039-.347.17-.445.345-.391.715-.543 1.503-.528 2.3.034 1.794.792 3.224 2.297 4.24.171.117.215.233.161.404-.103.35-.225.69-.332 1.041-.068.224-.171.272-.411.175-.826-.345-1.54-.856-2.17-1.473-1.07-1.036-2.038-2.178-3.246-3.073a13.77 13.77 0 0 0-.86-.588c-1.232-1.196.161-2.178.484-2.295.337-.122.117-.54-.973-.535-1.09.005-2.087.37-3.358.856a3.83 3.83 0 0 1-.582.17 11.953 11.953 0 0 0-3.602-.127C5.032 5.67 3.15 6.784 1.767 8.685.105 10.97-.286 13.567.193 16.275c.503 2.854 1.96 5.217 4.199 7.065 2.322 1.916 4.996 2.854 8.046 2.674 1.853-.107 3.915-.355 6.242-2.324.587.292 1.202.408 2.224.496.787.073 1.545-.039 2.131-.16.919-.195.855-1.045.523-1.201-2.309-.808-2.64-1.157-2.64-1.157 1.369-1.619 3.431-3.302 4.238-8.752.064-.433.01-.705 0-1.055-.005-.214.044-.297.288-.321a5.228 5.228 0 0 0 1.931-.593c1.745-.953 2.449-2.519 2.615-4.396.025-.288-.004-.584-.307-.735zM14.476 22.708c-2.61-2.052-3.876-2.728-4.399-2.699-.489.029-.401.588-.293.953.112.36.259.608.464.924.142.209.24.52-.142.754-.841.52-2.302-.175-2.371-.209-1.701-1.002-3.123-2.324-4.125-4.133-.968-1.741-1.53-3.608-1.623-5.602-.024-.481.117-.652.596-.739a5.87 5.87 0 0 1 1.911-.049c2.664.389 4.932 1.58 6.833 3.467 1.085 1.075 1.906 2.358 2.752 3.613.899 1.332 1.867 2.601 3.099 3.642.435.365.782.642 1.114.846-1.001.112-2.672.137-3.816-.768zm1.251-8.047a.383.383 0 0 1 .518-.36.379.379 0 0 1 .25.359.383.383 0 0 1-.386.384.378.378 0 0 1-.382-.383zm3.886 1.994c-.249.102-.499.19-.738.199-.371.019-.777-.131-.997-.316-.342-.287-.587-.447-.689-.948-.044-.214-.02-.545.02-.734.088-.408-.01-.671-.298-.909-.235-.195-.533-.248-.86-.248a.71.71 0 0 1-.318-.097c-.137-.068-.249-.238-.142-.447.034-.068.2-.233.24-.263.445-.253.958-.17 1.432.019.44.18.772.511 1.251.977.489.564.577.72.855 1.143.22.331.42.671.557 1.06.083.243-.024.442-.313.564z"/></svg>
```

Comment:
size can be even smaller
https://uploads.adguard.org/slqHnEiHY3LLVVOIBF.jpg

Reply:
Even with `1` is ok.